hack/jenkins/test-dockerized.sh

```diff
@@ -36,23 +36,20 @@ export PATH=${GOPATH}/bin:${PWD}/third_party/etcd:/usr/local/go/bin:${PATH}
 retry go get github.com/tools/godep && godep version
 retry go get github.com/jstemmer/go-junit-report
 
-# Enable the Go race detector.
-export KUBE_RACE=-race
-# Disable coverage report
-export KUBE_COVER="n"
-# Produce a JUnit-style XML test report for Jenkins.
-export KUBE_JUNIT_REPORT_DIR=${WORKSPACE}/artifacts
 export ARTIFACTS_DIR=${WORKSPACE}/artifacts
-# Save the verbose stdout as well.
-export KUBE_KEEP_VERBOSE_TEST_OUTPUT=y
-export KUBE_TIMEOUT='-timeout 300s'
+export KUBE_COVER="n"  # Disable coverage report
 export KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
+export KUBE_JUNIT_REPORT_DIR=${WORKSPACE}/artifacts  # junix xml files
+export KUBE_KEEP_VERBOSE_TEST_OUTPUT=y  # Save the verbose stdout as well.
+export KUBE_RACE=-race  # Enable the Go race detector.
+export KUBE_TEST_API_VERSIONS="${KUBE_TEST_API_VERSIONS:-v1}"
```

Comment:
if you feel this strongly about it being in source control in this file, please
* open an issue to revert this change and refer to it in a comment
* update `hack/jenkins/gotest-dockerized.sh` to pass in the Jenkins job name environment variable
* guard overriding the defaults so that it only occurs on the PR Jenkins unit/integration job